### PR TITLE
GuideBox: Fix network showing `null`, Change "paid" to show in button

### PR DESCRIPTION
**Pull request by MrChrisW**
Fixing #1475 & #1476

**Result**
selection_155

//CC @moollaza

### CODE REVIEW

**Comment by jagtalon** [comment]
I kinda don't like the look of it, but yeah would like to hear their comments.

**Comment by moollaza** [comment]
@MrChrisW I spoke with the design team and we decided that because the button takes you through to Guidebox's website where it indicates the viewing options and explains which ones are paid, we can just change our button text to "viewing options" and remove the "paid content" text

Sound good?

File: share/spice/guidebox/getid/guidebox_getid.js [hunk 1]
```diff
@@ -75,7 +78,7 @@
                     var abstract = Handlebars.helpers.ellipsis(item.overview, abstract_length);
```
Comment (moollaza) [inline share/spice/guidebox/getid/guidebox_getid.js]: One more thing, let's just stick with a shorter abstract `175` or `200` should be fine for both mobile and desktop.

**Comment by MrChrisW** [comment]
@moollaza Updated, changes visible here 
http://duckpan1.mrchris.me/?q=watch+edward+snowden+interview&ia=tv

File: share/spice/guidebox/getid/guidebox_getid.js [hunk 2]
```diff
@@ -67,15 +67,16 @@
...
-                                + " on "+ metadata.network;
+                    var network;
+                    metadata.network ? network = " on "+ metadata.network : network = "";
+                    
+                    var subtitle_tile = "Season "+ item.season_number+ ", #" + item.episode_number,
+                        subtitle_detail = "(Season "+ item.season_number+ ", #" + item.episode_number+")",
+                        abstract_length = (isMobile ? 175 : 200),
+                        abstract = Handlebars.helpers.ellipsis(item.overview, abstract_length),
+                        aired = "Originally aired "+ Handlebars.helpers.guideBox_getDate(item.first_aired)
+                                + network
```
Comment (jagtalon) [inline share/spice/guidebox/getid/guidebox_getid.js]: semicolon here?

Comment (MrChrisW) [inline share/spice/guidebox/getid/guidebox_getid.js]: As you wish!

**Comment by jagtalon** [comment]
:+1: noice

**Comment by MrChrisW** [comment]
@jagtalon [noiiice](http://imgur.com/gallery/ienS1)

**Comment by jagtalon** [comment]
@MrChrisW [noice](https://www.youtube.com/watch?v=rQnYi3z56RE)

**Comment by MrChrisW** [comment]
@jagtalon 
hahaha

**Pull request merged**